src/plugin/tokenHelpers.ts

```diff
@@ -0,0 +1,66 @@
+import {appendTypeToToken} from '@/app/components/createTokenObj';
+import {getAliasValue} from '@/utils/aliases';
+import checkIfAlias from '@/utils/checkIfAlias';
+import {SingleTokenObject} from 'Types/tokens';
+import {mergeDeep} from './helpers';
+
+export function findAllAliases(tokens) {
+    return tokens.filter((token) => checkIfAlias(token, tokens));
+}
+
+export function reduceToValues(tokens) {
+    const reducedTokens = Object.entries(tokens).reduce((prev, group) => {
+        prev.push({[group[0]]: group[1].values});
```

Comment:
```suggestion
        prev.push({[group[0]]: group[1]});
```
Due to the type change, this returns an error when updating json